Dockerfile

```diff
@@ -1,22 +1,17 @@
-FROM golang:1.13 as bd
+FROM golang:1.13 as build-env
```

Comment:
Should we upgrade the `golang` version to `1.15`? @kumarabd

Reply:
1.15 is the new default version, as documented.

Reply:
Good catch, @Aisuko